README.md

```diff
@@ -2,7 +2,7 @@
 
 Node.js API Client for the [Okta Platform API].
 
-Requires Node.js version 4.8.3 or higher.
+Requires Node.js version 6.9.0 or higher.
```

Comment:
do we need to be more specific in .travis.yml?

Reply:
Good call, I'm adding this